README.rst

```diff
@@ -1,76 +1,51 @@
 
 Introduction
 ------------
 
-This repository is a template for Open edX micro-frontend applications. It is flagged as a Template Repository, meaning it can be used as a basis for new GitHub repositories by clicking the green "Use this template" button above.  The rest of this document describes how to work with your new micro-frontend after you've created a new repository from the template.
+A tool used by course teams to communicate with thier learners. The interface for anything related to instructor to learner communications. Instructor bulk email, for example.
 
-After Copying The Template
---------------------------
 
-You'll want to do a find-and-replace to replace all instances of ``frontend-app-communications`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
-
-**Prerequisite**
-
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
-
-**Installation and Startup**
-
-In the following steps, replace "frontend-app-communications' with the name of the repo you created when copying this template above.
-
-1. Clone your new repo:
-
-  ``git clone https://github.com/edx/frontend-app-communications.git``
-
-2. Use node v12.x.
-
-   The micro-frontend build scripts support node 12.  Using other major versions of node *may* work, but is unsupported.  For convenience, this repository includes an .nvmrc file to help in setting the correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
-
-3. Install npm dependencies:
+Getting started
+------------
 
-  ``cd frontend-app-communications && npm install``
+For now, this repo is not intergrated with devstack. You'll be running the app locally and not through docker. This does make setup a little easier.
 
-4. Update the application port to use for local development:
+1. Clone the repo into your usual workspace
 
-   Default port is 8080. If this does not work for you, update the line `PORT=8080` to your port in all .env.* files
+   .. code-block::
 
-5. Start the dev server:
+      mkdir -p ~/workspace/
+      cd ~/workspace/
+      git clone https://github.com/edx/devstack.git
 
-  ``npm start``
+2. Install frontend dependencies
 
-The dev server is running at `http://localhost:8080 <http://localhost:8080>`_ or whatever port you setup.
+   .. code-block::
 
-Making Your New Project's README File
--------------------------------------
+      npm i
 
-Move the file ``README-template-frontend-app.rst`` to your project's ``README.rst`` file. Please fill out all
-the sections - this helps out all developers understand your MFE, how to install it, and how to use it.
+3. start the devserver. The app will be running at `localhost:8080`, or whatever port you change it too.
```

Comment:
Should we settle on a dedicated port now? If not, should we make a subtask to make sure we don't forget to do this? That way we won't have collisions with other MFE's once they get spun up?

Reply:
I thought I saw a list somewhere of the ports used by various MFEs... trying to find that now. I know I stumbled across it when we were creating the learner-record over the summer.

Reply:
Ah, yes, here are a few I know that are in use right now...
```
    'localhost:1997',  # frontend-app-account
    'localhost:1976',  # frontend-app-program-console
    'localhost:1994',  # frontend-app-gradebook
    'localhost:2000',  # frontend-app-learning
    'localhost:2001',  # frontend-app-course-authoring
    'localhost:3001',  # frontend-app-library-authoring
```

The learner-record MFE uses `1990` as well. Som anything that isn't being used?

Reply:
`1984` it is